components/tab/src/tab/tab.js

```diff
@@ -89,20 +86,30 @@ const Tab = ({
             }
 
             button:hover {
-                background-color: ${colors.grey100};
+                color: ${colors.grey900};
+            }
+
+            button:hover::after {
+                background-color: ${colors.grey600};
+                height: 2px;
             }
 
-            button:active {
-                /* Briefly highlight clicked tab */
-                background-color: ${colors.grey200};
+            button:active::after {
+                background-color: ${colors.grey800};
             }
 
             button.selected {
-                color: ${theme.primary700};
+                color: ${theme.primary800};
```

Comment:
I'm not sure I understand making the text `theme.primary800`, but the bar underneath `theme.primary700`? If that is a design choice, good by me.

Reply:
Yup, that's intentional. Text looks slightly lighter than blocks of color. Having the text a shade darker keeps it from looking too light next to the color block underneath.